src/pages/index.js

```diff
@@ -66,14 +46,22 @@ const addProfilePopup = new PopupWithForm(
   "#edit-profile",
   handleProfileFormSubmit
 );
+
+const avatarEditPopup = new PopupWithForm(
+  "#avatar-edit-modal",
+  handleAvatarSaveButton
+);
+
 const userInfo = new UserInfo({
   name: profileTitle,
   subtitle: profileSubtitle,
+  link: profileAvatar,
 });
 const cardImagePopup = new PopupWithImage(
   "#preview-image-modal",
   ".modal-preview-image"
 );
+
 //instantiate FormValidator class
 const addNewCardFormValidator = new FormValidator(
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```